config/routes.rb

```diff
@@ -30,6 +30,7 @@
       end
     end
     get '', to: 'companies#index'
+    devise_for :users
```

Comment:
Why is it required here when it is already present on line 46?

Reply:
resolved